Module-10/Task1.JSONSerialization/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Text.Json;
+using Task1.JSONSerialization;
+
+var department = new Department
+{
+    DepartmentName = "Sample department 1 ",
+    Employees = new List<Employee>
+    {
+        new() { EmployeeName = "SD-1 Employee 1" },
```

Comment:
you don't need to fix this, but in general it is better to use more real-world-like data to make your code easier to understand (especially when it's test code).
So a better EmployeeName would be "Jack Daniels" or "Frodo Baggins" or whatever other name you can come up with